ownCloudSDK/Core/Sharing/OCCore+Sharing.m

```diff
@@ -435,19 +436,19 @@ - (nullable NSProgress *)makeDecisionOnShare:(OCShare *)share accept:(BOOL)accep
 				// - drives
 				[[OCShareRole alloc] initWithType:OCShareRoleTypeManager
 						       shareTypes:OCShareTypesMaskUserShare|OCShareTypesMaskGroupShare
-						     permissions:OCSharePermissionsMaskRead|OCSharePermissionsMaskUpdate|OCSharePermissionsMaskCreate|OCSharePermissionsMaskDelete|OCSharePermissionsMaskShare
+						     permissions:OCSharePermissionsMaskRead|OCSharePermissionsMaskUpdate|OCSharePermissionsMaskCreate|OCSharePermissionsMaskDelete|(resharingSupported ? OCSharePermissionsMaskShare : 0)
 					 customizablePermissions:OCSharePermissionsMaskNone
 						       locations:OCLocationTypeDrive
 						      symbolName:@"person.fill"
 						   localizedName:OCLocalized(@"Manager")
-					    localizedDescription:OCLocalized(@"Upload, edit, delete, download, preview and share")],
+					    localizedDescription:OCLocalized(resharingSupported ? @"Upload, edit, delete, download, preview and share" : @"Upload, edit, delete, download and preview")],
 
```

Comment:
`OCLocalized` needs to be changed to `OCLocalizedString` to get in sync with the new translation changes before merging.

Reply:
Fixed.